pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -473,35 +502,40 @@ func describeDeploymentStatus(deploy *kapi.ReplicationController, first bool) st
 			reason = fmt.Sprintf(": %s", reason)
 		}
 		// TODO: encode fail time in the rc
-		return fmt.Sprintf("#%d deployment failed %s ago%s%s", version, timeAt, reason, describeDeploymentPodSummaryInline(deploy, false))
+		return fmt.Sprintf("#%d deployment failed %s ago%s%s", version, timeAt, reason, describePodSummaryInline(deploy, false))
 	case deployapi.DeploymentStatusComplete:
 		// TODO: pod status output
-		return fmt.Sprintf("#%d deployed %s ago%s", version, timeAt, describeDeploymentPodSummaryInline(deploy, first))
+		return fmt.Sprintf("#%d deployed %s ago%s", version, timeAt, describePodSummaryInline(deploy, first))
 	case deployapi.DeploymentStatusRunning:
-		return fmt.Sprintf("#%d deployment running for %s%s", version, timeAt, describeDeploymentPodSummaryInline(deploy, false))
+		return fmt.Sprintf("#%d deployment running for %s%s", version, timeAt, describePodSummaryInline(deploy, false))
 	default:
-		return fmt.Sprintf("#%d deployment %s %s ago%s", version, strings.ToLower(string(status)), timeAt, describeDeploymentPodSummaryInline(deploy, false))
+		return fmt.Sprintf("#%d deployment %s %s ago%s", version, strings.ToLower(string(status)), timeAt, describePodSummaryInline(deploy, false))
 	}
 }
 
-func describeDeploymentPodSummaryInline(deploy *kapi.ReplicationController, includeEmpty bool) string {
-	s := describeDeploymentPodSummary(deploy, includeEmpty)
+func describeRCStatus(rc *kapi.ReplicationController) string {
+	timeAt := strings.ToLower(formatRelativeTime(rc.CreationTimestamp.Time))
+	return fmt.Sprintf("rc/%s created %s ago%s", rc.Name, timeAt, describePodSummaryInline(rc, false))
+}
+
+func describePodSummaryInline(rc *kapi.ReplicationController, includeEmpty bool) string {
+	s := describePodSummary(rc, includeEmpty)
 	if len(s) == 0 {
 		return s
 	}
 	change := ""
-	desired := deploy.Spec.Replicas
+	desired := rc.Spec.Replicas
 	switch {
-	case desired < deploy.Status.Replicas:
+	case desired < rc.Status.Replicas:
 		change = fmt.Sprintf(" reducing to %d", desired)
-	case desired > deploy.Status.Replicas:
+	case desired > rc.Status.Replicas:
```

Comment:
> Trying to think if there could be any scenario where this could get stale (not "growing" in fact).

None comes to mind.  If you want more pods than you have, you're growing.